tox.ini

```diff
@@ -14,6 +14,7 @@ basedeps = mock>=1.3.0
 deps = {[testenv]basedeps}
        django
        keyring
+       jsonpickle
```

Comment:
they won't have any particular reason to have it. what do you have in mind? not totally sure how extra dependencies work.